benchmark/url/url-searchparams-update.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  searchParams: ['true', 'false'],
+  property: ['pathname', 'search'],
+  n: [1e6],
+});
+
+function main({ searchParams, property, n }) {
+  const url = new URL('https://nodejs.org');
+  if (searchParams === 'true') url.searchParams; // eslint-disable-line no-unused-expressions
```

Comment:
I recommend having an assert here.

Reply:
What would you assert here, just that `url.searchParams` is returning a `URLSearchParams`?

Reply:
Calling assert removes dead code elimination by the JIT